CMakeLists.txt

```diff
@@ -102,7 +103,15 @@ if (BUILD_DEPS)
     add_subdirectory(crt/aws-c-common)
 
     if (UNIX AND NOT APPLE AND NOT BYO_CRYPTO)
-        include(crt/aws-c-cal/cmake/modules/aws-lc.cmake)
+        if (ANDROID)
+            set(DISABLE_GO ON)
```

Comment:
I think we ALWAYS want to disable go, we never want to transitively create that dependency via CRT.